rc/extra/clang.kak

```diff
@@ -134,13 +134,13 @@ define-command -hidden clang-show-error-info %{
     evaluate-commands %sh{
         eval "set -- ${kak_opt_clang_errors}"
         shift # skip timestamp
-        for error in "$@"; do
+        desc=$(for error in "$@"; do
             if [ "${error%%|*}" = "$kak_cursor_line" ]; then
-                desc=$(printf '%s%s\n' "$desc" "${error##*|}")
+                printf '%s\n' "${error##*|}"
             fi
-        done
+        done)
         if [ -n "$desc" ]; then
-            printf %s\\n "info -anchor ${kak_cursor_line}.${kak_cursor_column} \"$desc\""
+            printf %s "${desc%x}" | sed "s/'/''/g" | xargs -0 printf "info -anchor %d.%d '%s'\n" "${kak_cursor_line}" "${kak_cursor_column}"
```

Comment:
I think that `%x` in `${desc%x}` is not required anymore, is it ?

Reply:
I'm also not sure `xargs -0` is needed here, this should probably be turned into a `while read -r;` loop.

Reply:
Is there any reason to avoid xargs? I thought it would be the clearest way to show we wanted the uninterpreted value as the argument.

Reply:
In this particular instance, the `-0` flag is not defined by POSIX so `xargs -0` is most likely not portable. Otherwise no strong bias against it!

Reply:
Oh, I didn't know the `-0` flag wasn't POSIX. I'll try and make a `while read` loop work.